app/views/placements/support/organisations/users/_details.html.erb

```diff
@@ -0,0 +1,24 @@
+<%# locals: (user: nil, organisation: nil) -%>
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l"><%= organisation.name %></span>
+      <h2 class="govuk-heading-l"><%= user.full_name %></h2>
+
+      <dl class="govuk-summary-list">
```

Comment:
same as above, if possible, could we use the summary_list helper?

Reply:
resolved in [853411c](https://github.com/DFE-Digital/itt-mentor-services/pull/125/commits/853411c3dc199496f425ee6adeb80fc5690ef163)